src/validation.cpp

```diff
@@ -2067,7 +2067,16 @@ bool ApplyTxInUndo(const CTxInUndo& undo, CCoinsViewCache& view, const COutPoint
         CCoinsModifier coins = view.ModifyCoins(out.hash);
         if (undo.nHeight != 0 ||
                 // Special-case genesis since nHeight is always 0, assume DB is clean
-                (Params().GenesisBlock().vtx[0]->GetHash() == out.hash && coins->IsPruned())) {
+                (Params().GenesisBlock().vtx[0]->GetHash() == out.hash && coins->IsPruned()) ||
```

Comment:
Actually this is redundant now. It can just be the one for loop checking all of them.

Reply:
We can probably get rid of this lambda too :)

Reply:
Couldn’t this still happen if we added more than one issuance in the genesis?

Reply:
I'm trying to say the code block below already covers this case, no?

Reply:
Ah yeah sure, i just thought the first time you mentioned it was in case we decided to change this code below

Reply:
Just in case :)

Reply:
I suppose it'll make rebasing easier.